pkg/apis/pipeline/v1beta1/container_types.go

```diff
@@ -238,6 +238,10 @@ type Step struct {
 type Ref struct {
 	// Name of the referenced step
 	Name string `json:"name,omitempty"`
+	// ResolverRef allows referencing a StepAction in a remote location
+	// like a git repo.
+	// +optional
+	ResolverRef `json:",omitempty"`
```

Comment:
missing json field?

Reply:
This is intentional, the same as taskref